uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-comments-tab.component.spec.js

```diff
@@ -85,10 +86,10 @@ define([
 
       it('adds comment with proper values', function () {
         expect(controller.request.comments[0]).toEqual({
-          contact_id: staffId,
+          contact_id: contactId,
           created_at: jasmine.any(String),
```

Comment:
So, it doesn't affect the order of just created comments so we can successfully remove it!